### ISSUE TITLE
G-Transformer+BERT setting 

### ISSUE THREAD

**Issue by WDZEthan** [baoguangsheng/g-transformer#3]
Hi author, can you share more details of the G-Transformer+BERT experimental settings? e.g., which version of the pre-trained BERT model (base or large, cased or uncased) was used, as well as the BERT decoding during inference to compute the s-BLEU scores. Thanks.

**Comment by baoguangsheng** [comment]
We use pretrained bert-base-multilingual-uncased model as the encoder to replace the encoder of a normal G-Transformer with following settings. The decoding is done by G-Transformer. Hope it can help. Thanks.

@register_model_architecture("transformer", "transformer_doc_bert")
def transformer_doc_bert(args):
    args.encoder_layers = getattr(args, "encoder_layers", 12)
    args.encoder_attention_heads = getattr(args, "encoder_attention_heads", 12)
    args.encoder_embed_dim = getattr(args, "encoder_embed_dim", 768)
    args.encoder_ffn_embed_dim = getattr(args, "encoder_ffn_embed_dim", 4*768)
    args.decoder_layers = getattr(args, "decoder_layers", 12)
    args.decoder_attention_heads = getattr(args, "decoder_attention_heads", 12)
    args.decoder_embed_dim = getattr(args, "decoder_embed_dim", 768)
    args.decoder_ffn_embed_dim = getattr(args, "decoder_ffn_embed_dim", 4*768)
    args.dropout = getattr(args, "dropout", 0.3)
    base_architecture(args)
